tests/test-run.sh

```diff
@@ -53,6 +53,10 @@ if ! $RUN true; then
     skip Seems like bwrap is not working at all. Maybe setuid is not working
 fi
 
+# Test help
+$BWRAP --help > help.txt
```

Comment:
I think you need ${BWRAP:-bwrap} here. Maybe we should just set BWRAP if its not already set.